src/test/java/org/stellar/sdk/responses/AccountsPageDeserializerTest.java

```diff
@@ -17,6 +19,16 @@ public void testDeserialize() {
     assertEquals(accountsPage.getLinks().getPrev().getHref(), "/accounts?order=desc&limit=10&cursor=1");
     assertEquals(accountsPage.getLinks().getSelf().getHref(), "/accounts?order=asc&limit=10&cursor=");
   }
+  
+  @Test
+  public void testDeserializeWithLiquidityPoolBalance() {
```

Comment:
great test coverage, do you think also worth adding one more check on regression aspect, i.e.:

`assertEquals(accountsPage.getRecords().get(0).getBalances()[1].getLiquidityPoolID(), Optional.absent());`

style-wise, good to remove the extra lines.